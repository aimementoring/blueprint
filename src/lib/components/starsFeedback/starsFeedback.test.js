import React from 'react';
import renderer from 'react-test-renderer';
import StarsFeedback from './starsFeedback';

describe('StarsFeedback', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(
        <StarsFeedback
          starSelected={0}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
