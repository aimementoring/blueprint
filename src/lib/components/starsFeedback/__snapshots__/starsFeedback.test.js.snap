// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StarsFeedback renders properly 1`] = `
<div
  className="rating "
>
  <span
    className=""
    onClick={[Function]}
  >
    ☆
  </span>
  <span
    className=""
    onClick={[Function]}
  >
    ☆
  </span>
  <span
    className=""
    onClick={[Function]}
  >
    ☆
  </span>
  <span
    className=""
    onClick={[Function]}
  >
    ☆
  </span>
  <span
    className=""
    onClick={[Function]}
  >
    ☆
  </span>
</div>
`;
